resolwe/toolkit/tools/parse_tabular_file.py

```diff
@@ -38,11 +36,10 @@ def main():
                         outline = line.strip().split(',')
                         csvwriter.writerow(outline)
             elif ext in ('.xls', '.xlsx'):
-                with open(args.input_file) as infile:
-                    workbook = xlrd.open_workbook(args.input_file)
-                    worksheet = workbook.sheets()[0]
-                    for rownum in range(worksheet.nrows):
-                        csvwriter.writerow(worksheet.row_values(rownum))
+                workbook = xlrd.open_workbook(args.input_file)
```

Comment:
I believe ```xlrd.open``` does not need closing as it isn't the same as ```open```
https://groups.google.com/forum/#!msg/python-excel/U2mKQkfLkGQ/juEAQsxlI-YJ